app/summit-2016/data/schedule.json

```diff
@@ -1,162 +1,142 @@
-        "speaker":"Zach Gibson, Yuin Chien, & Addy Osmani",
-        "time":"Tuesday @ 1:30pm",
-        "video": "https://www.youtube.com/watch?v=YiiXKoLYLMo&index=8&list=PLNYkxOF6rcICdISJclfQhj2S8QZGjXV8J",
-        "desc":"A look into Material Design's latest thinking around adaptive user interface design patterns and best practices. Polymer's paper elements make it easy to design cross form factor."
+        "name": "PWA's: Heavy Duty meets High Performance",
+        "speaker": "Chris Joel",
+        "desc": "You've done it. You've survived long enough as a web developer to witness the dawn of progressive web apps. Web apps now share the home screen with native apps, and have many of the same capabilities. But, with great power comes great user expectations. How do you achieve the delightful 60FPS view transitions on that native app your user just installed? How do you manage the burden of heavy image assets when every byte counts? How do you avoid jank when your app needs to do _real_ work? The Web Platform and Polymer Elements are here to help. In this session, we'll cover accessible strategies for delightful UX by leveraging the powerful primitives of the web. You will learn from real-world examples of heavy duty web app workloads. You will discover how to build and analyze these features to achieve a satisfying, high performance UX. And most importantly: you will learn how to make these features progressive, so that users across all browsers and devices can benefit from your work."
       },
       {
-        "name":"Polymer's Styling System",
-        "speaker":"Monica Dinculescu",
-        "time":"Tuesday @ 2:00pm",
-        "video": "https://www.youtube.com/watch?v=IbOaJwqLgog&index=9&list=PLNYkxOF6rcICdISJclfQhj2S8QZGjXV8J",
-        "desc":"Traditionally, CSS stylesheets have a terrible opinion about style encapsulation: the only thing stopping one element’s style from stomping over another’s is a “well chosen class name”. The Shadow DOM fixes this by building a little castle and a moat around each element, so that styles can’t get in and out of the web component castle. But what happens if you _do_ want to style this custom element? How do you cross the moat? Custom properties, that’s how."
+        "name": "An app in 3 days or: how I learned to stop worrying and love the Polymer toolbox",
+        "speaker": "Monica Dinculescu",
+        "desc": "Think of your favorite app. It's probably meowni.ca/emojillate (who can blame you), but if it isn't (and we need to talk) it probably has one, if not all, of these things: a database; a potential for disaster, with many users looking at and clicking on the same data; offline support, a responsive layout while preferably being faster than the average bear. Recreating this experience from scratch might sound like a terrifying experience, but I'm going to show you how the polymer toolbox and a few other already-built elements can help you go from zero to hero and build a PWA in no time."
```

Comment:
Second sentence is a run-on and quite hard to read: 

"It's probably meowni.ca/emojillate (who can blame you), but if it isn't (and we need to talk) it probably has one, if not all, of these things: a database; a potential for disaster, with many users looking at and clicking on the same data; offline support, a responsive layout while preferably being faster than the average bear."

Recommend fixing the list so it has an end, and making it two sentences:

"... offline support; and a responsive layout. While preferably being faster than the average bear."

Or add performance to the list. "offline support; better than average performance; and a responsive layout." (Admittedly, that cuts one joke.)

Second sentence:
"Recreating this experience from scratch might sound like a terrifying experience, but I'm going to show you how the polymer toolbox and a few other already-built elements can help you go from zero to hero and build a PWA in no time."

"experience" is repeated—just dropping the first experience sounds fine to me ("Recreating this from scratch"), but "Recreating this app from scratch" or "Creating an app like this from scratch" would also work. "polymer toolbox" should be "Polymer App Toolbox."